sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosBatchItemRequestOptions.java

```diff
@@ -22,6 +22,9 @@ public CosmosBatchItemRequestOptions() {
 
     /**
      * Gets the If-Match (ETag) associated with the operation in CosmosBatch.
+     * Most commonly used with replace and delete requests.
```

Comment:
Do we want to be more explicit that when doing `Upsert` for a document that exists then `Replace` semantics are followed if `If-Match` header is set and `Create` semantics when the document doesn't exist?

Reply:
We can, the reason I didn't add it is because not all of these options work for upserts (for example, containers and databases can't be upserted). 
However, I can clarify this on item, batch, bulk and other request options which support upsert functionality.

Reply:
Done.